01-node-tutorial/answers/07-mind-grenade.js

```diff
@@ -0,0 +1,8 @@
+const num1 = 10
+const num2 = 5
+
+function DivideNumbers() {
+  console.log(`10 divided by 5 :  {num1 / num2}  `)
```

Comment:
Running this file prints the following
```
10 divided by 5 :  {num1 / num2} 
```

Reply:
I updated the code. Let me know if it works now.